dGame/dComponents/CharacterComponent.cpp

```diff
@@ -46,6 +46,9 @@ CharacterComponent::CharacterComponent(Entity* parent, Character* character) : C
 	if (LandingAnimDisabled(character->GetZoneID()) || LandingAnimDisabled(Game::server->GetZoneID()) || m_LastRocketConfig.empty()) {
 		m_IsLanding = false; //Don't make us land on VE/minigames lol
 	}
+    auto tableData = CDClientDatabase::ExecuteQuery("SELECT LevelCap FROM WorldConfig WHERE WorldConfigID = 1 LIMIT 1;");
+    m_MaxLevel = tableData.getIntField(0, -1);
+    tableData.finalize();
```

Comment:
This query can be cached somewhere like dZoneManager or as a static member.

Reply:
Could you clarify what you mean by dZoneManager?  Is there a specific file this should be stored in?  

Could you also expand on what you mean by 'as a static member'?

Reply:
I am currently caching this result in dZoneManager.cpp.  If this should be changed I will do so.